SYCL/KernelAndProgram/spec_constants_after_link.cpp

```diff
@@ -3,6 +3,9 @@
 // RUN: %GPU_RUN_PLACEHOLDER %t.out
 // FIXME: ACC devices use emulation path, which is not yet supported
 
+// HIP backend does not currently implement linking.
+// XFAIL: hip
```

Comment:
```suggestion
// UNSUPPORTED: hip
```
?

Reply:
I'm not against going with UNSUPPORTED, but since this is a consistent failure due to missing functionality, XFAIL has the added benefit of telling us when the test is ready to be enabled, so we don't miss enabling it. It does of course come with the downside of needing a merge of both test-suite changes and intel/llvm changes being merged at the same time, causing the problem with failing tests in the test-suite CI.